gems/quilt_rails/CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 <!-- ## [Unreleased] -->
 
+### Changed
+
+- Remove automatic passing of `X-CSRF-Token` in the header. With [csrf header strategy](./README.md#fixing-rejected-csrf-tokens-for new-user-sessions) you should not need this value for GraphQL request. If absolutely needed, use [custom headers method](./README.md#example:-sending-custom-headers-from-rails-controller) to pass the value manually. ([#1509](https://github.com/Shopify/quilt/pull/1509))
```

Comment:
[csrf header strategy](./README.md#fixing-rejected-csrf-tokens-for new-user-sessions) isn't resolving the hyperlink.

Reply:
https://github.com/Shopify/quilt/tree/master/gems/quilt_rails/README.md#fixing-rejected-csrf-tokens-for-new-user-sessions is what I was aiming for...there must be a typo in there...

Reply:
fixed!